### DISCUSSION TITLE
Inpainting question

### DISCUSSION THREAD

**Discussion by E2GO** [mashb1t/Fooocus#83] [Q&A] [answered]
First of all I understand that Fooocus active development is finished but I have a small problem so may be someone could redirect me to how to solve it.

In some scenarios, I have an image with a resolution of, for example, 1024x1024. However, I need to enhance the finer details in it so that they become less blurry and appear sharper. The automatic mask size for inpainting adjusts in such a way that it's difficult to add these details. In Automatic1111, for instance, I simply increase the mask size, and then all the necessary details are added in good quality.

So, my question is: in the advanced settings, I couldn’t find parameters that directly affect this. Is it possible to add this feature to the interface or the mask creation algorithm, allowing for better control?

Thank you.

**Comment by mashb1t** [comment, marked as answer]
Hey @E2GO, Fooocus already does an internal resizing of the inpaint area to 1024x1024ish resolution, no matter how small you inpaint, then scales the inpainted image down to the original size to match your input image size. This depends on the inpaint mode.

If this doesn't work for you you can use another fork with photo editor support like photopea or a free photo editor, save your image in 2x or more resolution without AI upscaling, then inpaint the area to get more area of the mask for inpainting in the final result.

I don't know the full list of features of other forks, but https://github.com/metercai/SimpleSDXL is the most advanced and is still maintained afaik.

**Reply by E2GO** [reply]
Oh I got your point! I already do this with Photoshop yes) Just wanted to do it more in automatic mode. 
Thanks for fast answer and suggestion. Gone digging it I suppose)
